### PR TITLE
Adding PyDrive2

### PR DESCRIPTION
**Pull request by efiop**
Hi @conda-forge/help-python ! Please take a look when you'll have time. Thank you! 🙏 

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by efiop** [comment]
@shcheklein Could you please confirm with a comment that you are agreeing to become a maintainer for this feedstock repo?

**Comment by shcheklein** [comment]
I confirm.

**Comment by efiop** [comment]
Just to clarify: PyDrive2 is an actively maintained fork of PyDrive, which hasn't released anything since 2016, and PRs for which are not getting actively merged/reviewed. We are actively maintaining PyDrive2 as we are actively using it in our [dvc project](https://github.com/iterative/dvc), which has feedstock here https://github.com/conda-forge/dvc-feedstock .

**Comment by antonkulaga** [comment]
PyDrive2 has a killer-feature - teamdrive support, please merge it!

File: recipes/pydrive2/LICENSE [hunk 1]
```diff
@@ -0,0 +1,185 @@
+Copyright 2013 Google Inc. All Rights Reserved.
```
Comment (isuruf) [inline recipes/pydrive2/LICENSE]: Can you delete this file? It is picked up from the source tarball

Comment (shcheklein) [reply recipes/pydrive2/LICENSE]: @isuruf done!

**Comment by efiop** [comment]
Thank you @isuruf ! 🙏

**Pull request merged**